code/modules/projectiles/guns/energy/special.dm

```diff
@@ -76,10 +76,11 @@
 	w_class = WEIGHT_CLASS_TINY
 
 /obj/item/gun/energy/mindflayer
-	name = "\improper Mind Flayer"
-	desc = "A prototype weapon recovered from the ruins of Research-Station Epsilon."
-	icon_state = "xray"
-	item_state = null
+	name = "mind flayer"
```

Comment:
why not improper?

Reply:
Improper when it was an unique gun, now it’s just a regular gun like the others.